src/angular/ghettobox/ghettobox.scss

```diff
@@ -0,0 +1,96 @@
+@import '../styles/common';
```

Comment:
focus indication with keyboard navigation could be improved, respectively made visible

Reply:
Remove tabindex
Remove outline on ghettobox